tests/suites/test_suite_aes.function

```diff
@@ -372,6 +371,33 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
```

Comment:
This could contain the information about dependency on the proper configuration option.